code/modules/mob/living/simple_animal/hostile/retaliate/icarus_drone.dm

```diff
@@ -0,0 +1,263 @@
+// friendly icarus combat drones
+/mob/living/simple_animal/hostile/retaliate/icarus_drone
```

Comment:
I'm not sure if I agree with the amount of copypaste here. A lot of this shit is already defined on malf drones. I'd say generalize the drone and then split it.